package.json

```diff
@@ -31,52 +31,55 @@
   },
   "types": "src/openmrs-esm-patient-chart.tsx",
   "devDependencies": {
-    "@babel/core": "^7.5.5",
-    "@babel/plugin-proposal-class-properties": "^7.5.5",
-    "@babel/preset-env": "^7.5.5",
-    "@babel/preset-react": "^7.0.0",
-    "@babel/preset-typescript": "^7.3.3",
+    "@babel/core": "^7.7.2",
+    "@babel/plugin-proposal-class-properties": "^7.7.0",
+    "@babel/preset-env": "^7.7.1",
+    "@babel/preset-react": "^7.7.0",
+    "@babel/preset-typescript": "^7.7.2",
     "@openmrs/esm-error-handling": "^1.1.0",
-    "@openmrs/react-root-decorator": "^1.0.0",
-    "@testing-library/react": "^9.1.3",
-    "@types/jest": "^24.0.18",
-    "@types/react": "^16.9.2",
-    "@types/react-dom": "^16.9.0",
-    "@types/react-router-dom": "^4.3.5",
-    "babel-eslint": "^10.0.2",
+    "@openmrs/react-root-decorator": "^3.1.0",
+    "@testing-library/react": "^9.3.2",
+    "@types/jest": "^24.0.23",
+    "@types/react": "^16.9.11",
+    "@types/react-dom": "^16.9.4",
+    "@types/react-router-dom": "^5.1.2",
+    "@types/webpack-env": "^1.14.1",
+    "babel-eslint": "^10.0.3",
     "babel-jest": "^24.9.0",
     "babel-loader": "^8.0.6",
     "browserslist-config-openmrs": "^1.0.0",
     "clean-webpack-plugin": "^3.0.0",
     "concurrently": "^5.0.0",
     "css-loader": "^3.2.0",
-    "dayjs": "^1.8.16",
-    "eslint": "^6.2.1",
-    "eslint-config-prettier": "^6.1.0",
-    "eslint-config-ts-react-important-stuff": "^1.0.0",
-    "eslint-plugin-prettier": "^3.1.0",
-    "fork-ts-checker-webpack-plugin": "^1.5.0",
-    "husky": "^3.0.4",
+    "dayjs": "^1.8.17",
+    "eslint": "^6.6.0",
+    "eslint-config-prettier": "^6.7.0",
+    "eslint-config-ts-react-important-stuff": "^1.1.0",
+    "eslint-plugin-prettier": "^3.1.1",
+    "fork-ts-checker-webpack-plugin": "^3.1.0",
+    "husky": "^3.1.0",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^24.9.0",
     "jest-cli": "^24.9.0",
-    "prettier": "^1.18.2",
-    "pretty-quick": "^1.11.1",
-    "react-router": "^5.0.1",
-    "react-router-dom": "^5.0.1",
-    "single-spa-react": "^2.10.2",
+    "prettier": "^1.19.1",
+    "pretty-quick": "^2.0.1",
+    "react-router": "^5.1.2",
+    "react-router-dom": "^5.1.2",
+    "single-spa-react": "^2.10.4",
     "style-loader": "^1.0.0",
-    "systemjs-webpack-interop": "^1.1.0",
-    "typescript": "^3.5.3",
-    "webpack": "^4.39.2",
-    "webpack-cli": "^3.3.7",
-    "webpack-dev-server": "^3.8.0"
+    "systemjs-webpack-interop": "^1.1.2",
+    "typescript": "^3.7.2",
+    "webpack": "^4.41.2",
+    "webpack-cli": "^3.3.10",
+    "webpack-dev-server": "^3.9.0"
   },
   "dependencies": {
-    "@openmrs/esm-api": "^1.0.8",
-    "@types/react-router": "^5.0.3",
-    "react": "^16.9.0",
-    "react-dom": "^16.9.0"
+    "@openmrs/esm-api": "^1.1.0",
+    "@types/react-router": "^5.1.3",
+    "i18next": "^19.0.1",
```

Comment:
the new version of openmrs-react-root-decorator has a peer dependency on i18next and react-i18next. See https://github.com/openmrs/openmrs-react-root-decorator/releases